pom.xml

```diff
@@ -51,7 +51,7 @@
     <jdk.min.version>1.8</jdk.min.version>
 
     <sonar.version>6.7.1</sonar.version>
-    <sonar-java.version>5.1.0.13090</sonar-java.version>
+    <sonar-java.version>5.6.1.15064</sonar-java.version>
```

Comment:
@KengoTODA 
Did you tested this version?

This could cause two small problems:
 - People deploying SonarQube automatically (Docker recipe or equivalent) will place the latest jar of sonar-fb .. but will not override the core jars because they are expected to be stable and tested. Causing plugin not loading because of missing dependency.
 - SonarQube some times added older version included in the official download: https://www.sonarqube.org/downloads/ This means somebody testing the next release on a brand new installation could fail. (This occurs in two previous release.)

For this reason, I try not to force the dependency update to core stuff because it can be in the way of many users.